pkg/products/grafana/reconciler.go

```diff
@@ -143,6 +143,12 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 		return phase, err
 	}
 
+	phase, err = r.reconcileGrafanaDashboards(ctx, client, "rate-limiting")
```

Comment:
Hey @laurafitzgerald do you think it would be better to make the name of dashboard a variable, just in case we would want to change it or add more dashboards later on?